jest.config.js

```diff
@@ -39,4 +39,5 @@ module.exports = {
   testEnvironmentOptions: {
     url: 'http://localhost/',
   },
+  testTimeout: 10000,
```

Comment:
I failed to find a better way of fixing the tests that were failing intermittently. This issue was also happening on my `main` branch. Reproduce the issue using this gitpod: https://gitpod.io#snapshot/b7d2e77d-2157-4f2c-bc3f-f74677ecbefa

Reply:
Should be obviated by #1573